### DISCUSSION TITLE
Discrepancies between sequencing runs

### DISCUSSION THREAD

**Discussion by bshim181** [milaboratory/mixcr#1787] [General]
Hello, 

I am analyzing a bulk TCR dataset with UMIs. 
From one sample aliquiot, we distributed it evenly into 4 to generate 4 sample replicates. 
We have sequenced these 4 replicates twice and compared the outputs from two sequencing runs. 
We are noticing significant discrepancies in clonal frequency between two sequencing runs on the same clone. 

For example for the clone with,
CAVTDSNYQLIW | TRAV1-2 | TRAJ33

UMI Fraction For Sequencing Run 1 
<br class="Apple-interchange-newline">
<img width="995" alt="Screenshot 2024-09-11 at 3 58 33 PM" src="https://github.com/user-attachments/assets/5211465f-623b-4a8e-992a-1f416a48d7df">

UMI Fraction For Sequencing Run 2

<img width="895" alt="Screenshot 2024-09-11 at 4 00 16 PM" src="https://github.com/user-attachments/assets/bbd45d66-a571-42a1-9e72-b70073a3c141">

I am wondering what might be causing these discrepancies between replicates and across sequencing runs. I am getting approximately similar alignment and assemble report statistics.

**Comment by mizraelson** [comment]
It’s hard to say, but clearly something is off during sample preparation. The numbers clearly show a tenfold difference.
